frontend/src/reducers/exerciseReducer.js

```diff
@@ -48,6 +49,12 @@ export default function reducer(
       };
     }
 
+    case 'exercise/SET_URL_WEBPAGE':
+      return {
+        ...state,
+        pageURL: action.payload.url,
+      };
+
```

Comment:
What is this for? Do we need it? We only use it here https://github.com/cybsafe/phishtray/pull/320/files#diff-815edb5fa8b84b3062e64c1962545036R52-R57